IFC4x3/Properties/e/Emodulus_1Bctvx1L570B_eZVJbTFB8/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Emodulus_1Bctvx1L570B_eZVJbTFB8" Name="Emodulus" UniqueId="4b9b7e7b-0551-4700-bfa8-8df4e574f2c8" PrimaryDataType="IfcModulusOfElasticityMeasure" />
```

Comment:
Use `YoungModulus`.

Reply:
Fixed in [78d1f16](https://github.com/bSI-InfraRoom/IFC-Specification/pull/597/commits/78d1f1672489922a8eb8773bdb382784e4071e42)